frontend/src/components/TopicCard/Follow.js

```diff
@@ -3,16 +3,17 @@ import PropTypes from 'prop-types'
 
 class Follow extends Component {
   render = () => {
-    const { isFollowing, onTopicFollow } = this.props
-    return <div className='topicFollow' onClick={onTopicFollow}>
-      {isFollowing ? 'Unfollow' : 'Follow'}
+    const { ActiveMapper, isFollowing, onTopicFollow } = this.props
+    return <div className='topicFollow' onClick={() => ActiveMapper && onTopicFollow()}>
+      {ActiveMapper ? isFollowing ? 'Unfollow' : 'Follow' : ''}
```

Comment:
this is too clever for me. I'd rather this be a function, honestly, with spelled out if statements. Even though the chained ternaries are pretty sexy lol

Reply:
yeah, will do. I rushed this line